src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx

```diff
@@ -41,9 +40,9 @@ const ArrayTemplateForm = (props) => {
   const renderHeading = () => {
     // FIXME: Should use Paragraph component of Typography
     if (element.children.props.formData.name) {
-      return <p>{heading}<Icon icon={isEditClicked ? faMinus : faPlus} /></p>;
+      return <div className="form-expander-row"><div className="form-expander-row__heading">{heading}</div><span className="form-expander-row__icon"><Icon icon={isEditClicked ? faMinus : faPlus} /><ArrayTemplateRemoveButton element={element} /></span></div>;
```

Comment:
Please follow class naming convention strictly.
```suggestion
      return <div className="array-template-form__form-expander-row"><div className="array-template-form__form-expander-row-heading">{heading}</div><span className="array-template-form__form-expander-row-icon"><Icon icon={isEditClicked ? faMinus : faPlus} /><ArrayTemplateRemoveButton element={element} /></span></div>;
```

Reply:
Done